tests/bindings/python/test_version.py

```diff
@@ -1,21 +1,22 @@
 #
 # Tests the version function bindings
 #
-import sys
 import unittest
 
+
 class VersionTestCase(unittest.TestCase):
 
-    def test_validator(self):
+    def test_version(self):
```

Comment:
So in the C++ tests we check that the correct version is returned - part of the testing when a version increment happens, I think :) Not sure if it matters that the python test also checks or if we're just checking that the method works. More just noting this here cause I saw it and I'm not sure which is better.

Reply:
In general* I've only tested if the method can be called without error, and if it accepts/returns the correct types. In other words, I've only checked that the method (seems to) map to the correct C++ function, leaving the testing of the underlying C++ code for the C++ tests!

*with a few exceptions here and there, for example, if I set a name I check if I can then get the name. I think a small bit of this kind of testing is necessary to check that the Python bindings actually have an effect (or if they're doing nothing but not raising errors)

Reply:
Sounds good.